prompting/tasks/task_registry.py

```diff
@@ -67,8 +69,7 @@ class TaskRegistry(BaseModel):
         # ),
         TaskConfig(
             task=WebRetrievalTask,
-            # TODO: Increase probability after v2.9.0.
-            probability=0.03,
+            probability=0.1,
```

Comment:
We actually cannot increase the rate of web retrieval because we get rate limited by the web retrieval on DDGS

Reply:
done!